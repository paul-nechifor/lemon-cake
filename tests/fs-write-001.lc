(fs-write 'wot' 'I am a file. What are you?')
(repr (fs-read 'wot'))
