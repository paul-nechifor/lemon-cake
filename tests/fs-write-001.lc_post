pwd
cd ..
rm -fr tmp_test_dir
