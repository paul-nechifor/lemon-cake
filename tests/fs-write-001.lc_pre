mkdir tmp_test_dir
cd tmp_test_dir
