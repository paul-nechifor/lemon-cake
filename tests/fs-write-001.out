'I am a file. What are you?'
